hazelcast/src/jepsen/hazelcast.clj

```diff
@@ -386,15 +388,24 @@
                            :workload checker})
             :model      model})))
 
+(defn test-names
+  "Comma separated list of available test"
+  []
+  (str/join ", "(map name (keys (tests nil)))))
+
+
 (def opt-spec
   "Additional command line options"
-  [[nil "--workload WORKLOAD" "Test workload to run, e.g. atomic-long-ids."
-    :parse-fn keyword]])
+  [[nil "--workload WORKLOAD" (str "Test workload to run, possible options are " (test-names) "Required.")
+    :parse-fn keyword
```

Comment:
May I suggest `:parse-fn (comp (tests) keyword)`, and `:validate [identity (jepsen.cli/one-of (tests))]`? Then you don't need any of the changes to jepsen.cli, and workload lookup is handled by the option parser for you.